virttest/qemu_vm.py

```diff
@@ -3895,7 +3912,7 @@ def activate_nic(self, nic_index_or_name):
             pcie = False
         else:
             pcie = nic['nic_model'] not in ['e1000', 'rtl8139']
-        bus_spec = self._get_pci_bus(nic_params, 'nic', pcie)
+        bus_spec = self._get_bus(nic_params, 'nic', pcie)
```

Comment:
Should we update `_get_pci_bus` to `_get_bus` in all places?

Reply:
I think we could update in cases in the future since I'm not sure how many cases will be influenced